pkg/controllers/user/rbac/handler_base.go

```diff
@@ -199,7 +195,8 @@ func (m *manager) createClusterRole(rt *v3.RoleTemplate) error {
 	logrus.Infof("Creating clusterRole for roleTemplate %v (%v).", rt.DisplayName, rt.Name)
 	_, err := m.clusterRoles.Create(&rbacv1.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: rt.Name,
+			Name:        rt.Name,
+			Annotations: map[string]string{clusterRoleOwner: rt.Name},
 		},
```

Comment:
v1.Roles are also created with the name of the roleTemplate, should we track them and delete when roleTemplate no longer exists by adding annotations for them too here? https://github.com/rancher/rancher/blob/master/pkg/controllers/management/auth/manager.go#L735

Reply:
Well, those roletemplates are getting created in management so they could just be an owner reference but I don't know if that is intended behavior and outside the scope of this PR.